scilpy/utils/streamlines.py

```diff
@@ -285,3 +286,48 @@ def compress_sft(sft, tol_error=0.01):
     compressed_sft.to_space(orig_space)
 
     return compressed_sft
+
+
+def cut_invalid_streamlines(sft):
+    """ Cut streamlines so their longest segment are within the bounding box """
```

Comment:
clean docstring

Reply:
should it be cut_invalid_points ?

Reply:
Well, technically, you are cutting streamlines, points are indivisible entities